lib/utils/index.js

```diff
@@ -579,6 +581,55 @@ class Utils {
       throw err;
     }
   }
+
+  static iosRealDeviceUDID(udid) {
+    if (udid.length > 25) {
+      return udid;
+    }  
+    
+    if (udid.length < 24) {
+      throw new Error('Incorrect UDID provided for real iOS device');
+    }
+
+    return `${udid.substring(0, 8)}-${udid.substring(9, 25)}`;
+  };
+
+  static getSdkRootFromEnv() {
+    const androidHomeInGlobalEnv = 'ANDROID_HOME' in process.env;
+
+    dotenv.config({path: path.join(__dirname, '.env')});
+
+    const androidHome = process.env.ANDROID_HOME;
+    const fromDotEnv = androidHomeInGlobalEnv ? '' : ' (taken from .env)';
+
+    if (androidHome) {
+      const androidHomeFinal = untildify(androidHome);
+
+      const androidHomeAbsolute = path.resolve(__dirname, androidHomeFinal);
+      if (androidHomeFinal !== androidHomeAbsolute) {
+        return androidHomeAbsolute;
+      }
+
+      return androidHomeFinal;
+    }
+
+    let err;
+
+    if (androidHome === undefined) {
+      err = new Error('ANDROID_HOME environment variable is NOT set!');
+    } else {
+      err = new Error(`ANDROID_HOME is set to '${androidHome}'${fromDotEnv} which is NOT a valid path!`);
+    }
+    
+    err.help = [
+      `Run command to setup ${Logger.colors.cyan('npx nightwatch-mobile-helper android --setup')}`,
```

Comment:
Can remove `--setup` from here.

Can change it to "Try running `npx @nightwatch/mobile-helper android`".